python/cuproj/README.md

```diff
@@ -0,0 +1,59 @@
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;cuSpatial - GPU-Accelerated Vector Geospatial Data Analysis</div>
```

Comment:
This README references cuSpatial. Might need some updates in this file.

Reply:
I have a separate PR for docs: #1237

Reply:
That PR depends on this one.